packages/tools/client-debug-view/src/Debugger.tsx

```diff
@@ -54,25 +61,39 @@ export function FluidClientDebugger(props: FluidClientDebuggerProps): React.Reac
 		};
 	}, [clientDebugger, setIsContainerDisposed]);
 
+
+	let view: React.ReactElement;
 	if (clientDebugger === undefined) {
-		return (
+		view = (
 			<NoDebuggerInstance
-				containerId={containerId}
-				onRetryDebugger={(): void => setClientDebugger(getFluidClientDebugger(containerId))}
+				containerId={"No container found"}
+				onRetryDebugger={(): void => setClientDebugger(getFirstDebugger())}
 			/>
 		);
-	}
-
-	if (isContainerDisposed) {
-		return (
+	} else if (isContainerDisposed) {
+		view = (
 			<DebuggerDisposed
-				containerId={containerId}
-				onRetryDebugger={(): void => setClientDebugger(getFluidClientDebugger(containerId))}
+				containerId={clientDebugger.containerId}
+				onRetryDebugger={(): void => setClientDebugger(getFirstDebugger())}
 			/>
 		);
+	} else {
+		view = <ClientDebugView containerId={clientDebugger.containerId} clientDebugger={clientDebugger} />;
 	}
 
-	return <ClientDebugView containerId={containerId} clientDebugger={clientDebugger} />;
+	return <div
```

Comment:
Be sure to run the `npm run format` script inside the client-debug-view package. I suspect prettier will want to make some changes here.

Reply:
Fixed.

Reply:
Thanks!